README.md

```diff
@@ -73,8 +73,10 @@ Also check [Before you install](https://docs.bigbluebutton.org/2.2/install.html#
 | `bbb_ntp_cron_day` | Day of the month the time-sync job should run | `*`
 | `bbb_ntp_cron_hour` | Hour when the time-sync job should run | `5`
 | `bbb_ntp_cron_minute` | Minute when the time-sync job should run | `0`
-| `bbb_html5_node_options` | Allow to set extra options for node for the html5-webclient | unset | Could be used for example with https://github.com/bigbluebutton/bigbluebutton/issues/11183 ; `--max-old-space-size=4096 --max_semi_space_size=128`
+| `bbb_html5_node_options` | Allow to set extra options for node for the html5-webclient | unset | Could be used for example with <https://github.com/bigbluebutton/bigbluebutton/issues/11183> ; `--max-old-space-size=4096 --max_semi_space_size=128`
 | `bbb_meeting_inactivity_timeout_minutes` | set the default timeout in minutes | `10` |
+| `bbb_freeswitch_socket_password` | set password for freeswitch |  | Can be generated with `pwgen -s 16 1` |
+| `bbb_multikurento` | enable multikurento setup | `false` |
```

Comment:
Since I did not test this yet I made the default `false`.
If this feature is stable and working and is beneficial for ever instance this could be default `true`